Test/BaseTestCase.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Taxjar\SalesTax\Test;
+
+use PHPUnit\Framework\TestCase;
+
+class BaseTestCase extends TestCase
+{
+    public function todo(string $message = '')
```

Comment:
I've been thinking about having this todo method in the BaseTestClass and I think its something we should talk about.

To me it feels similar to having a to do comment. i.e. //TODO:

I think an argument can be made for to do comments and for this method. However most of the time, it seems like eventually their original context or details get forgotten about but the to do comment remains. Future developers then may be hesitant to remove them as they know someone must have though they were important at some point. I think we would be better served by removing this and using our project management tools to track coverage we would like to add. What do you think?

Reply:
I can get behind using `// TODO: ...` comments. I do think marking the test as skipped helps with visibility when devs are forced to observe yellow test suite pass instead of green, but I also agree that using the current project management tools should be the source of truth for tracking these issues and coverage gaps. As such, I'll remove this functionality.